smithery.yaml

```diff
@@ -0,0 +1,17 @@
+# Smithery configuration file: https://smithery.ai/docs/config#smitheryyaml
+
+startCommand:
+  type: stdio
+  configSchema:
+    # JSON Schema defining the configuration options for the MCP.
+    type: object
+    required:
+      - minecraftContainerName
+    properties:
+      minecraftContainerName:
+        type: string
+        description: The Docker container name for the running Minecraft server.
+  commandFunction:
+    # A function that produces the CLI command to start the MCP on stdio.
+    |-
+    (config) => ({ command: 'python', args: ['rcon.py'], env: { 'MINECRAFT_CONTAINER_NAME': config.minecraftContainerName } })
```

Comment:
You'll need to adapt the code to this new environment variable as well. It's currently hardcoded to `mc`. Same with the README.